lib/services/local/users.go

```diff
@@ -1470,13 +1470,12 @@ func (s *IdentityService) GetKeyAttestationData(ctx context.Context, publicKey c
 		return nil, trace.BadParameter("missing parameter publicKey")
 	}
 
-	sshPub, err := ssh.NewPublicKey(publicKey)
+	keyFingerprint, err := attestedKeyFingerprint(publicKey)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
 
-	key := ssh.FingerprintSHA256(sshPub)
-	item, err := s.Get(ctx, backend.Key(attestationsPrefix, key))
+	item, err := s.Get(ctx, backend.Key(attestationsPrefix, keyFingerprint))
```

Comment:
@Joerger, refresh my memory, is key attestation already out?

What are the implications of changing the storage key at this stage?

Should we try to read the "old" key as well, for a few versions, as a fallback?

Reply:
Yes, changing this to be URL bas64 encoded will be backwards incompatible. This amounts to the user needing to relogin, as their previous login certs won't find validated attestation data. I agree that we should check for the old key as a fallback for now.